public/data/Nnadivictory25.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "Nnadi  Victory",
+    "type": "personal",
+    "bio": "Aspiring Fullstack Developer!",
+    "avatar": "https://github.com/Nnadivictory25.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Nnadivictory25",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/NnadiVictory01",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Coding",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

Thanks for this